pkgdb2/default_config.py

```diff
@@ -52,6 +52,9 @@
 # List the ACL which are auto-approved (don't need reviewing)
 AUTO_APPROVE = ['watchcommits', 'watchbugzilla']
 
+### Blacklisted items
+REQUEST_BLACKLIST = ['fedora']
```

Comment:
We probably want to make this a dict allowing to specify blacklisted package per namespace